package.json

```diff
@@ -1,50 +1,50 @@
 {
-  "name": "@axelar-network/axelar-cgp-solidity",
-  "version": "4.5.0",
-  "description": "EVM Smart Contracts for Axelar Network",
-  "main": "index.js",
-  "scripts": {
-    "build": "rm -rf artifacts && hardhat compile",
-    "test": "hardhat test",
-    "test-evm-versions": "bash test-evm-versions.bash",
-    "lint": "solhint 'contracts/**/*.sol' && eslint 'test/**/*.js'",
-    "prettier": "prettier --write 'contracts/**/*.sol' 'test/**/*.js' 'scripts/**/*.js' '*.js'",
-    "flatten": "sh scripts/flatten-contracts.sh"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/axelarnetwork/axelar-cgp-solidity.git"
-  },
-  "keywords": [
-    "ethereum",
-    "axelar"
-  ],
-  "author": "axelar-network",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/axelarnetwork/axelar-cgp-solidity/issues"
-  },
-  "homepage": "https://github.com/axelarnetwork/axelar-cgp-solidity#readme",
-  "dependencies": {
-    "@axelar-network/axelar-gmp-sdk-solidity": "^3.3.0"
-  },
-  "devDependencies": {
-    "@nomicfoundation/hardhat-toolbox": "^2.0.2",
-    "chai": "^4.3.7",
-    "dotenv": "^16.0.1",
-    "eslint": "^8.36.0",
-    "eslint-config-richardpringle": "^2.0.0",
-    "ethers": "^5.7.2",
-    "fs-extra": "^11.1.1",
-    "hardhat": "^2.13.0",
-    "lodash": "^4.17.21",
-    "mocha": "^10.2.0",
-    "prettier": "^2.8.7",
-    "prettier-plugin-solidity": "1.0.0-beta.19",
-    "readline-sync": "^1.4.10",
-    "solhint": "^3.4.1"
-  },
-  "engines": {
-    "node": "^16.0.0 || ^18.0.0"
-  }
-}
\ No newline at end of file
+    "name": "@axelar-network/axelar-cgp-solidity",
```

Comment:
we decided to use 2 space indentation for JSON files. Can you update your editor config? And add an entry in prettier config for json?

Reply:
Updated prettier config